src/evict/evict_lru.c

```diff
@@ -1494,6 +1494,7 @@ __wt_cache_wait(WT_SESSION_IMPL *session, int full)
 
 		switch (ret = __wt_evict_lru_page(session, 0)) {
 		case 0:
+			cache->app_evicts++;
```

Comment:
Another way to express my confusion with the calculation is that I don't see where this is being read.